scripts/generate-readmes.js

```diff
@@ -1,13 +1,19 @@
 const {readFileSync} = require('fs');
 const {renderFile} = require('template-mate');
+const YAML = require('yaml');
 
 // eslint-disable-next-line max-len
 const DO_NOT_EDIT_PREFIX = '<!-- DO NOT EDIT THIS FILE! THIS FILE WAS AUTOGENERATED BY TEMPLATE-MATE -->\n<!-- SEE https://github.com/angus-c/just/blob/master/CONTRIBUTING.md#readme-template -->\n\n';
 
-const packageVariables = JSON.parse(
-  readFileSync('./md-variables.json', 'utf8')
+const packageVariables = YAML.parse(
+  readFileSync('./packages.yml', 'utf8')
 );
 
+Object.entries(packageVariables).forEach(([name, data]) => {
+  packageVariables[name].examples = data.examples.trim();
+  packageVariables[name].description = data.description.trim();
```

Comment:
Descriptions and examples will have a trailing newline without this!

Reply:
Closed by [`f9d537f` (#569)](https://github.com/angus-c/just/pull/569/commits/f9d537fedb3a5be01bd234eff16328a114372de8). Uses the "-" [block chomping indicator](https://yaml.org/spec/1.2.2/#8112-block-chomping-indicator) to strip final/trailing newlines instead of stripping them after parsing.